### ISSUE TITLE
Clarification and expansion of example usage

### ISSUE THREAD

**Issue by JSchmie** [RAI-SCC/torch_blue#51]
Hi,

As part of the [JOSS review](https://github.com/openjournals/joss-reviews/issues/9415), and as you already know from #49, I checked out your demo.

One of the JOSS review criteria is:

> Example usage: Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).

From my perspective, the current [tutorial file](https://github.com/RAI-SCC/torch_blue/blob/main/scripts/pytorch_tutorial.py) does not fully satisfy this requirement.

You do explain a lot of information in the API reference, which is very helpful, but it would be even more useful (at least for me as a user) if you could more explicitly show what changes in the *outputs* when using your library. One concrete suggestion would be to create a Jupyter notebook that documents your current demo, with some additional explanation.

In particular, in the evaluation at the end, you could highlight the advantage of drawing samples in a Bayesian Neural Network (BNN), and show how this allows you to quantify the uncertainty of a prediction.

**Comment by JSchmie** [comment]
Thank you, this look s fine to me!
